.github/workflows/clp-core-build.yaml

```
@@ -201,17 +208,37 @@ jobs:
         run: "chown $(id -u):$(id -g) -R ."
         shell: "bash"
 
-      - uses: "./.github/actions/clp-core-build"
-        env:
-          OS_NAME: "ubuntu-jammy"
+      - uses: "./.github/actions/clp-core-image-run"
         with:
           image_name: "${{env.DEPS_IMAGE_NAME_PREFIX}}${{env.OS_NAME}}"
-          use_shared_libs: "${{matrix.use_shared_libs}}"
           use_published_image: >-
```

actionlint (1.7.4)

property "ubuntu_focal_image_changed" is not defined in object type {centos_stream_9_image_changed: string; clp_changed: string; ubuntu_jammy_image_changed: string}

(expression)